sql/expression_value.h

```diff
@@ -225,6 +225,10 @@ struct ExpressionValueInfo {
     /// a row.
     virtual std::vector<KnownColumn> getKnownColumns() const;
 
+    /// Return the set of known atoms for a row.  Default throws that it's not
```

Comment:
I don't think this should be here.  It does type analysis, and so should be in ExpressionValueInfo.